presto-orc/src/main/java/com/facebook/presto/orc/writer/ByteColumnWriter.java

```diff
@@ -66,13 +67,14 @@
 
     private boolean closed;
 
-    public ByteColumnWriter(int column, Type type, CompressionParameters compressionParameters, Optional<DwrfDataEncryptor> dwrfEncryptor, MetadataWriter metadataWriter)
+    public ByteColumnWriter(int column, int sequence, Type type, CompressionParameters compressionParameters, Optional<DwrfDataEncryptor> dwrfEncryptor, MetadataWriter metadataWriter)
     {
         checkArgument(column >= 0, "column is negative");
         requireNonNull(compressionParameters, "compressionParameters is null");
         requireNonNull(dwrfEncryptor, "dwrfEncryptor is null");
         requireNonNull(metadataWriter, "metadataWriter is null");
         this.column = column;
+        this.sequence = sequence;
```

Comment:
Please add a check for the sequence parameter. checkArgument (sequence >= 0, "sequence is negative")

Reply:
Same for all the other places as well.